test cases/cmake/25 assembler/meson.build

```diff
@@ -0,0 +1,9 @@
+project('assembler test', ['c'])
```

Comment:
You also need 'cpp' here, so that meson can tell CMake the correct C++ compiler.

Alternatively, only enable `c` in CMake by explicitly setting the languages in the CMake `project()` call.